### PR TITLE
feat: add door handler

### PR DESCRIPTION
**Pull request by tkhmy**
- add door service for opening the door
- add publish door status to `/vehicle/status/door_status`
- The pacmod version will need to update
```bash
sudo apt install ros-galactic-pacmod3==2002.1.0-0focal
```

### CODE REVIEW

**Comment by tkhmy** [comment]
@tkimura4 , Hi can you review this PR when you free?
together with this PR
https://github.com/tier4/tier4_ad_api_adaptor/pull/23

**Review by tkimura4 — DISMISSED** [review]
The code seems to be good.
(I have not confirmed that door operation works properly on the actual vehicle. )

**Comment by tkhmy** [comment]
Thanks
TODO: test this in real vehicle before merge @tkhmy

File: pacmod_interface/src/pacmod_interface/pacmod_interface.cpp [hunk 1]
```diff
@@ -128,6 +131,8 @@ PacmodInterface::PacmodInterface()
     create_publisher<pacmod3_msgs::msg::SystemCmdInt>("/pacmod/shift_cmd", rclcpp::QoS{1});
   turn_cmd_pub_ =
     create_publisher<pacmod3_msgs::msg::SystemCmdInt>("/pacmod/turn_cmd", rclcpp::QoS{1});
+  door_cmd_pub_ = create_publisher<pacmod3_msgs::msg::SystemCmdInt>(
+    "pacmod/as_rx/rear_pass_door_cmd", rclcpp::QoS{1});
```
Comment (tkhmy) [inline pacmod_interface/src/pacmod_interface/pacmod_interface.cpp]:
```suggestion
    "/pacmod/as_rx/rear_pass_door_cmd", rclcpp::QoS{1});
```

**Comment by tkhmy** [comment]
@tkimura4 Hi. I confirmed the changes work with the pacmo3 latest version (2002.1.0-0focal), can you review this PR?

**Comment by tkimura4** [comment]
Memo: 
After the PR is merged, the pacmod_interface will not work if the version of pacmod < `2002.1.0-0focal`.

**Review by tkimura4 — APPROVED** [review]
LGTM

**Pull request merged**